0800-appendix-198236-trib-to-kitwanga.Rmd

```diff
@@ -49,12 +49,14 @@ fpr::fpr_table_wshd_sum(site_id = my_site) %>%
 
 ```
 
+<br>
+
+This tributary joins with the Kitwanga River before draining into Kitwanga Lake. This lake has an important value to fisheries in the area. Using salmon enumeration facilities to trap and count upstream migrating salmon, the Gitanyow Fisheries Authorities have proven that Kitwanga Lake is one of the most productive sockeye nursery lakes for its size in all of BC [@gitanyowfisheriesauthority2023KitwangaRiver]. The Kitwanga River salmon enumeration facility is located approximately 4km north of the confluence with the Skeena river. This facility provides counts of five species of salmon: sockeye, chinook, coho, chum, and pink. In response to rapid declines in sockeye stocks in recent years, the Kitwanga Lake Sockeye Restoration project was started in 2006.  Restoration efforts include: environmental monitoring of the lake, assessment of sockeye spawning and rearing habitat, and hatchery rearing [@gottesfeld_etal2009CaseHistory]. 
```

Comment:
"In response to rapid declines in sockeye stocks in recent years, the Kitwanga Lake Sockeye Restoration project was started in 2006"

17 years ago is recent?!

the Kitwanga Lake Sockeye Restoration project? Website says "Kitwanga River Sockeye Recovery Plan"

colons in the middle of a sentence are strange.  Sometime they are used before a line breaks when bullets follow but...

Reply:
![image](https://user-images.githubusercontent.com/99455166/229185565-5366c85f-ae53-41bf-85e9-40b15f999321.png)